src/deye_inverter_state.py

```diff
@@ -101,3 +105,12 @@ def __is_device_observation_changed(self, events: DeyeEventList) -> bool:
         self.__event_updated = time.time()
         self.__last_observations = events
         return True
+
+    def __is_data_ready(self, observation_events: list[DeyeObservationEvent]) -> bool:
+        readiness_check_observations = [
+            event.observation for event in observation_events if event.observation.sensor.is_readiness_check
+        ]
+        self.__log.debug(f"Data readiness observations: {readiness_check_observations}")
```

Comment:
What do you think about changing the log logic here so that only not ready data is reported?

This would lead to less log content, as only unexpected statuses would be logged.

Reply:
I will keep it as it is for two reasons:
1. `readiness_check_observations` contains zero or one element (total energy). 
2. it's logged at debug level, so it will not appear in the logs when the default INFO log level is configured